marklogic-data-hub/src/main/java/com/marklogic/hub/flow/FlowRunner.java

```diff
@@ -23,18 +23,81 @@
 
 public interface FlowRunner {
 
+    /**
+     * Sets the flow to be used with the flow runner
+     * @param flow the flow object to be used
+     * @return the flow runner object
+     */
     FlowRunner withFlow(Flow flow);
+
+    /**
+     * Sets the batch size for the flow runner
+     * @param batchSize - the size of the batch in integer form
+     * @return the flow runner object
+     */
     FlowRunner withBatchSize(int batchSize);
+
+    /**
+     * Sets the thread count for the flowrunner
+     * @param threadCount - the number of threads for the flow runner to use
+     * @return the flow runner object
+     */
     FlowRunner withThreadCount(int threadCount);
+
+    /**
+     * Sets the source client on the flow runner
```

Comment:
Sets the source client on the flow runner. The source client determines which database to run against for building the envelope.